imageio/imageio-pict/src/main/java/com/twelvemonkeys/imageio/plugins/pict/PICTImageReader.java

```diff
@@ -392,6 +392,9 @@ private void readPICTopcodes(ImageInputStream pStream) throws IOException {
                     }
                     opCode = pStream.readUnsignedShort();
                 }
+                if (DEBUG) {
+                    System.out.printf("opCode 0x%04x at pos %04x :\n", opCode, pStream.getStreamPosition()-2);
```

Comment:
As said, let me know if you want me to send the code style settings. It's a lot easier if we get it right and can save a lot of roundtrips... 😀 

Anyway, always spaces around operators. 

Also, I don't think 4 digits is enough for positions in large files, that's why I suggested `%08x` . 😀 

```suggestion
                    System.out.printf("opCode 0x%04x at pos %08x:\n", opCode, pStream.getStreamPosition() - 2);
```

PS: Note that you can apply suggestions directly from the Github UI.

Reply:
Code style settings won't help... you're assuming I'm using the same IDE as you :-) I use Eclipse for writing code (and my own personal code style that I strictly adhere to!) but I don't have a git plugin for that as I normally use svn. So... I'm painfully copying over over and reapplying changes one by one into BBEdit, and then using the command line to commit to git!